Code/lib/corner/corner.cpp

```diff
@@ -0,0 +1,90 @@
+#include <corner.h>
+
+CornerMerchantry::CornerMerchantry(int trigPin_1, int echoPin_1, int trigPin_2, int echoPin_2, float *right_threshold, float *front_threshold)
+{
+    set_threshold_right = right_threshold;
+    set_threshold_front = front_threshold;
+    this->trigPin_front = trigPin_1;
+    this->echoPin_front = echoPin_1;
+    this->trigPin_side = trigPin_2;
+    this->echoPin_side = echoPin_2;
+    pinMode(trigPin_front, OUTPUT);
```

Comment:
Set the on another function let's say init that is called during setup and not when instantiating the class. Check out here https://github.com/0804robot/mazerunner/pull/12#discussion_r979464966